R/read_from_dhis2-helpers.R

```diff
@@ -166,3 +168,42 @@ dhis2_get_attributes_from_user <- function(args_list) {
     end_date           = end_date
   )
 }
+
+#' Make full URL from request attributes
+#'
+#' @param base_url the web address of the server the user wishes to log in to
+#' @param dataset a vector or a list of comma-separated data set identifiers
+#' @param org_unit a vector or a list of comma-separated organisation
+#'    unit identifiers
+#' @param start_date the start date for the time span of the values to export
+#' @param end_date the end date for the time span of the values to export
+#'
+#' @return a character string with the full URL to be used in the API request
+#' @keywords internal
+#'
+dhis2_make_request_url <- function(base_url,
```

Comment:
Let's keep using httr/httr2 dedicated built-in functionality for this rather than pasting.

See these functions to pass a list of query parameters: https://httr2.r-lib.org/reference/req_body.html